src/store.js

```diff
@@ -68,7 +72,8 @@ class Store {
    * Binds an action type => handler
    */
   on(actionType, handler) {
-    this.__handlers = this.__handlers.set(actionType, handler)
+    console.assert(actionType, "Store.on called without actionType");
```

Comment:
not too comfortable with using `console.assert` here

I think it'd be find to throw an error.

I will probably add a `__DEV__` define in webpack in the near future and we can wrap these types of invariants in that flag that will compile out in prod bundles.